api/terraform/modules/catalogue_api_stack/service/main.tf

```diff
@@ -31,18 +43,20 @@ module "task_definition" {
     apm_secret     = "catalogue/api/apm_secret"
   }
 
-  sidecar_container_image = var.nginx_container_image
-  sidecar_container_port  = var.nginx_container_port
-  sidecar_container_name  = local.sidecar_container_name
-  sidecar_cpu             = 512
-  sidecar_memory          = 1024
+  sidecar_name   = local.sidecar_container_name
+  sidecar_image  = var.nginx_container_image
```

Comment:
You should use the nginx module for this if you can, so as to keep a consistent image for API Gateway backed services across all services.